app/src/main/java/com/example/nvdovin/weatherapp/utils/Executor.java

```diff
@@ -29,6 +31,13 @@ public void execute() throws Exception {
             final City city = retrofitFactory.getData(cityName);
             greenDaoFactory.insert(city);
         }
-        EventBus.getDefault().post(greenDaoFactory.loadCities());
+
+        SortQueryBuilder sortQueryBuilder = new SortQueryBuilder();
+        sortQueryBuilder.setAscending(true);
+        sortQueryBuilder.setProperty(CityDao.Properties.Name);
+        SortQueryBuilder[] builders = new SortQueryBuilder[]{
```

Comment:
no need for an array when using varargs. find some time to read varargs module in Java